plugin-api/src/main/java/net/clgd/ccemux/util/OperatingSystem.java

```diff
@@ -31,4 +37,16 @@ else if (name.startsWith("Mac"))
 		else
 			return Other;
 	}
+
+	public static boolean isWindows() {
+		return get().equals(Windows);

```

Comment:
I realise this is old code, but if we're following Java conventions the enums should probably be `WINDOWS`, `MAC_OSX`, etc... Seeing as this is an enum, we can also get away with `== Windows` instead, which is a little easier to read.